### PR TITLE
prefix wp table with wp_

### LINKED ISSUE #4696: [Bug] Wordpress installer fails when generated table prefix starts with a number
### Describe the bug

Tables are not allowed to start with a number. So when the random generator accidentally generated a prefix that starts with a number the installation fails. To avoid this we should actually just prefix the prefix with `wp` ;).

Related line:
https://github.com/hestiacp/hestiacp/blob/dbba10c701799010eb22e0745a78b125311e4394/web/src/app/WebApp/Installers/WordPress/WordPressSetup.php#L85

### Tell us how to replicate the bug

1. Run WordPress installer and be the lucky person to generate a prefix with an number.

### Which components are affected by this bug?

Control Panel Web Interface

### Hestia Control Panel Version

1.8.12

### Operating system

Debian 12

### Log capture

_No response_

### PR DESCRIPTION
**Pull request by divinity76**
resolves #4696

### CODE REVIEW

File: web/src/app/WebApp/Installers/WordPress/WordPressSetup.php [hunk 1]
```diff
@@ -79,11 +79,11 @@ public function install(array $options = null) {
...
 		);
 		foreach ($result->raw as $line_num => $line) {
-			if ('$table_prefix =' === substr($line, 0, 15)) {
+			if (str_starts_with($line, '$table_prefix =')) {
 				$result->raw[$line_num] =
-					'$table_prefix = \'' .
-					addcslashes(Util::generate_string(5, false) . "_", "\\'") .
-					"';\r\n";
+					'$table_prefix = ' .
+					var_export("wp_" . Util::generate_string(5, false) . "_", true) .
```
Comment (rjd22) [inline web/src/app/WebApp/Installers/WordPress/WordPressSetup.php]: Nitpick: We could clean this up by using a bit more proper function for string formatting:

```
$result->raw[$line_num] = sprintf(
    '$table_prefix = "wp_%s_";' . "\r\n",
    Util::generate_string(5, false),
);
```

Comment (divinity76) [reply web/src/app/WebApp/Installers/WordPress/WordPressSetup.php]: i'd rather use var_export, it is *_the_* correct way to export strings to php source code.
For example, if generate_string was to return 
```
foo'bar
```
var_export would know that the ' needs escaping and return
```
'foo\'bar'
```
while sprintf would not.

Could change it to
```
$result->raw[$line_num] = sprintf(
    '$table_prefix = %s;' . "\r\n",
    var_export("wp_".Util::generate_string(5, false)."_",true),
);
```

Comment (rjd22) [reply web/src/app/WebApp/Installers/WordPress/WordPressSetup.php]: Okay, the above sample given by you would indeed be more clearer.

Comment (divinity76) [reply web/src/app/WebApp/Installers/WordPress/WordPressSetup.php]: @rjd22 code updated

**Review by jaapmarcus — APPROVED** [review]

**Pull request merged**